File: algorithm-boj/week1/2576.py
sum, min = 0, 100

for _ in range(7):
    n = int(input())
    if n % 2 != 0:
        if n < min:
            min = n
        sum += n

if sum == 0:
    print(-1)
else:
    print(sum)
    print(min)

File: minseok/2576/2576/2576.cpp
#include <iostream>

using namespace std;

int main() {
	int input[7];
	int sum = 0, min = 500, cnt=0;

	for (int i = 0; i < 7; i++) {
		cin >> input[i];
		if (input[i] % 2 == 1) {
			sum += input[i];
			if (input[i] < min) {
				min = input[i];
				cnt++;
			}
		}
	}
	if (cnt == 0)
		cout << "-1";
	else
		cout << sum << endl << min;
}